package.json

```diff
@@ -39,11 +45,13 @@
   },
   "homepage": "https://github.com/asciidoctor/asciidoctor-reveal.js",
   "dependencies": {
+    "@asciidoctor/cli": "^3.1.0",
```

Comment:
Isn't this a problem in the overall objective of allowing users to run their own asciidoctor.js version? Globally installed asciidoctor.js will not conflict or cause trouble?

Reply:
The package `@asciidoctor/cli` does not have a direct dependency on `@asciidoctor/core`: https://github.com/asciidoctor/asciidoctor-cli.js/blob/7055e675d429ad6b9b0b57bf395d383a0cc6b6a3/package.json#L51-L53